### ISSUE TITLE
[🐞] Global navigation bar appearance updates after web fall back

### ISSUE THREAD

**Issue by quintonpryce** [paypal/paypalcheckout-ios#100]
### 🐞 Describe the Bug
After seeing the web fallback flow the global navigation bar appearance is reset.

### 🔬 Minimal Reproduction

1) Apply custom navigation bar appearance presets to remove lines/shadows & to make it translucent.
2) Try to make a checkout & get the web fall back flow.
3) Observe the entire app's navigation bars.

Example
```
            UINavigationBar.appearance().setBackgroundImage(UIImage(), for: .default)
            UINavigationBar.appearance().shadowImage = UIImage()
            UINavigationBar.appearance().backgroundColor = .clear
            UINavigationBar.appearance().isTranslucent = true
            UINavigationBar.appearance().tintColor = .white
            UINavigationBar.appearance().titleTextAttributes = [.foregroundColor: UIColor.white]
            UINavigationBar.appearance().largeTitleTextAttributes = [.foregroundColor: UIColor.white]
            UINavigationBar.appearance().prefersLargeTitles = true
            UINavigationBar.appearance().barStyle = .black
```

### 😕 Actual Behavior
The navigations bar's appearance is being reset.

### 🤔 Expected Behavior
The navigation bar's appearance to remain unaffected.

### 🌍 Environment

- iOS version: - 15.4
- SDK version (`Checkout.version`): - 0.88

### ➕ Additional Context

https://user-images.githubusercontent.com/15935475/164736564-8284dd72-8b7f-4e6f-841f-444824e0c394.mov

**Comment by jonathajones** [comment]
Hey @quintonpryce  - 
Thanks for opening the issue and including reproduction steps. We'll triage and fix on our end.

**Comment by jonathajones** [comment]
This has been resolved in the latest release (0.94.0)
